sympy/geometry/ellipse.py

```diff
@@ -1005,6 +1005,45 @@ def semilatus_rectum(self):
         """
         return self.major * (1 - self.eccentricity ** 2)
 
+    def auxiliary_circle(self):
+        """Returns equation of auxiliary circle of the ellipse.
+
+        Auxiliary Circle of an ellipse is circumcircle of an ellipse i.e. the circle
+        whose center concurs with that of the ellipse and whose radius is equal to the ellipse's semi-major axis.
+
+        Parameters
+        ==========
+
+        x : str, optional
+            Label for the x-axis. Default value is 'x'.
+        y : str, optional
+            Label for the y-axis. Default value is 'y'.
```

Comment:
These parameters are not there any more so this should be removed from the docstring.

Reply:
yup these parameters are not needed at present forget to delete it thank you @oscarbenjamin